scripts/proteinInteractionEBI/parseEBI.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import collections
+import re
+import sys
+
+
+def get_class_name_helper(astring):
+    """Convert a name string to format: ThisIsAnUnusualName.
+    
+    Take a space delimited string, return a class name such as ThisIsAnUnusualName 
+    Here we use this function for instance name. Thus it allows to start with a number
+    """
+    jointName = astring.title().replace(' ','')
+    # substitute except for  _, character, number
+    nonLegit = re.compile(r'[\W]+')
+    className = nonLegit.sub('', jointName)
+    return className
+
+def get_class_name(astring):
+    return get_class_name_helper(astring)
```

Comment:
Why do you need two functions here?

Reply:
Sure I will delete the unnecessary functions.